Error handling fails in presence of incremental MH transforms

For example:

```js
Infer({
  method: 'incrementalMH',
  model() {
    assert.ok(false, 'fail');
  }});
```

Fails with:

```
TypeError: Cannot read property 'line' of undefined
```

... rather than showing:

```
AssertionError: fail
    at err.wppl:49

48|   model() {
49|     assert.ok(false, 'fail');
    -----------^
50|   },
```

I understand the problem and will submit a fix soon.

(Thanks @mhtess for pointing this out to me.)

--- a/src/errors/errors.js
+++ b/src/errors/errors.js
@@ -27,7 +27,7 @@ function addressToWpplCallStack(address, addressMap) {
   var stack = addressToStack(address);
   return _.chain(stack).map(function(id) {
     var loc = addressMap[id];
-    return loc && {
+    return loc && loc.start && {
       fileName: loc.source,
       lineNumber: loc.start.line,
       columnNumber: loc.start.column,

--- a/tests/test-stack-trace.js
+++ b/tests/test-stack-trace.js
@@ -172,6 +172,15 @@ var testDefs = [
             {line: 11, col: 9, name: 'f'},
             {line: 10, col: 0, name: 'Enumerate'}],
     debug: true
+  },
+
+  { name: 'with caching transform',
+    code: ["Infer({method: 'incrementalMH',",
+           '  model: function() {',
+           '  assert.ok(false);',
+           '}});'].join('\n'),
+    stack: [{line: 3, col: 9, name: 'ok'}],
+    debug: true
   }
 ];
 

